kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SettingUpKolibri.vue

```diff
@@ -1,7 +1,28 @@
 <template>
 
   <div class="full-page">
-    <main class="content">
+    <div
+      v-if="coreError"
+      style="padding: 16px;"
+    >
+      <AppError :hideParagraphs="true">
+        <h2>{{ coreError }}</h2>
+        <template
```

Comment:
Glad you caught this! I put this in `OnboardingStepBase` not fully considering that `SettingUpKolibri` doesn't use it 🤦🏻‍♂️